packages/rsk3-utils/src/index.js

```diff
@@ -1046,6 +1050,81 @@ const soliditySha3 = function() {
     return keccak256(`0x${hexArguments.join('')}`);
 };
 
+/**
+ * Gets the RSK address from BTC private key
+ *
+ * @method getRskAddress
+ *
+ * @param {String} btcPrivateKey BTC Private key.
+
+ * @return {String} RSK address.
+ */
+const getRskAddress = function(btcPrivateKey) {
+    const myWallet = wallet.fromPrivateKey(Buffer.from(_keyBtcToRskInBytes(btcPrivateKey)));
+    const addressInRskFormat = myWallet.getAddress();
+    return addressInRskFormat.toString('hex');
+};
+
+/**
+ * Get RSK Private Key from BTC Private Key
+ *
+ * @param {String} btcPrivateKey BTC Private key.
+ *
+ * @return {String} RSK Private Key.
+ */
+function getRskPrivateKey(btcPrivateKey) {
```

Comment:
```suggestion
const getRskPrivateKey = function (btcPrivateKey) {
```

for consistency with the other functions